process:
  path_loss_file: "pathLoss.csv"
  GP_file: "GP"
  minimum_rss_value: -95
  sub_folder: "/28-02-22/" # Specifying to pick data from which date
  minimum_threshold: -65 # Only rss value above this value will be counted
  minimum_repetition_factor: 0.3 # float, between 0 to 1 